### ISSUE TITLE
free space on sshfs mounted NAS drives shown wrong

### ISSUE THREAD

**Issue by graygrotesque** [doublecmd/doublecmd#552]
doublecmd shows me a weird number of free space on my NAS drive, while dolphin for example gets it right.

there should be 1,3 tib free but doublecmd shows 348 T free. Is this configurable / how does dc calculate these numbers?

this is how dc shows free space
![dc](https://user-images.githubusercontent.com/97067711/185594675-471093b6-7e67-4da4-946c-4687b1366589.png)

this is how dolphin shows it (this is correct)
![dolphin](https://user-images.githubusercontent.com/97067711/185594812-9d1dca67-7456-4a98-979e-409cfa06dbea.png)

**Comment by alexx2000** [comment]
No, DC use standard [statfs ](https://linux.die.net/man/2/statfs)function to get free space.

**Comment by rich2014** [comment]
may be a similar bug in FPC RTL (but linux) #554
